README.md

```diff
@@ -28,7 +28,7 @@ sudo apt-get install g++ python-dev
 
 Windows users need a Python installation and C++ compiler:
 
-* Install some version of Python (2.7 or higher), e.g., [Anaconda](https://www.anaconda.com/distribution/) is a good choice for users not yet familiar with the language
+* Install some version of Python 2 (2.7 or higher), e.g., [Anaconda](https://www.anaconda.com/distribution/) is a good choice for users not yet familiar with the language
```

Comment:
2.7 is the last Python2 version so this can be changed to `Install Python 2.7 version`